openIMIS/openIMIS/settings.py

```diff
@@ -225,8 +224,11 @@ def SITE_URL():
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "core.middleware.SecurityHeadersMiddleware",
 ]
 
+MODE = os.environ.get("MODE", "DEV")
```

Comment:
I don't think we can force this decision by us. Please check it with @delcroip if we can set DEV mode as default or revert this change.

Reply:
We can keep just MODE without setting up default one.